pkg/platform/docker/docker.go

```diff
@@ -29,15 +25,16 @@ type Docker struct {
 	Name        string
 	IsSwarm     bool
 	Client      *client.Client
+	Config      config.StaticConfig
 }
 
-func NewDocker(cnf platforming.Config) (d *Docker, err error) {
-	d = &Docker{NetworkName: cnf.Docker.Network, Name: cnf.Docker.Name}
-	if cnf.Use == "swarm" { //https://github.com/openbaton/go-docker-vnfm/blob/8d0a99b48e57d4b94fa14cdb377abe07eaa6c0aa/handler/docker_utils.go#L113
+func NewDocker(cfg config.StaticConfig) (d *Docker, err error) {
```

Comment:
lets pass platforming.Config, and not the entire static config

Reply:
We need to pass it so docker can customize the config and encode it.

Reply:
pkg/platform/util/config.go

Reply:
Ah, you absolutely right about this one. I am wondering if there is a better way to go about it. to pass platforming.Config, yet somehow fetch the entire config in a different way.